### PR TITLE
Optimisations

### PR DESCRIPTION
**Pull request by mkholjuraev**
This removes loading Inventory module from InventoryConnector. Instead it loads the module directly into the component. Also there is some cleanup as well. 

The metrics are varying across runs, but I am getting average numbers. Here you can see the effect of this optimisation work. Most improved metric is LCP which has increased from ~7.5 seconds to ~ 4 seconds. 

Befor the work: 
withoutOptimizationCommit

After the work:
withOptimizationCommit

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -112,3 +113,17 @@ export const INVENTORY_WRITE_PERMISSIONS = [
     'inventory:hosts:write',
     'inventory:*:write'
 ];
+
+export const getSearchParams = () => {
```
Comment (mkholjuraev) [inline src/constants.js:117]: If I had time, I would do a larger refactoring of filtering in the Inventory App. but for now, this is bearable.

File: src/Routes.js [hunk 2]
```diff
@@ -21,25 +22,19 @@ export const Routes = () => {
...
-                        rhcdFilter={searchParams.getAll(RHCD_FILTER_KEY)}
-                        updateMethodFilter={searchParams.getAll(UPDATE_METHOD_KEY)}
-                        page={searchParams.getAll('page')}
-                        perPage={searchParams.getAll('per_page')}
-                    />}
+                    render={() =>
+                        <RenderWrapper
+                            cmp={InventoryTable}
+                            isRbacEnabled
+                            {...getSearchParams()}
```
Comment (bastilian) [inline src/Routes.js:36]: It might make sense to wrap this in a `useMemo`.

Comment (mkholjuraev) [reply src/Routes.js:36]: Good point!

**Comment by mkholjuraev** [comment]
Closing the PR for the sake of already merged PR.

**Pull request closed without merging**